client/src/components/editor/LinkAnet.js

```diff
@@ -33,14 +38,25 @@ const LinkAnet = ({ entityKey, contentState, url }) => {
     )
   } else {
     // Non ANET entity link
-    return <>{urlLink}</>
+    return (
+      <a href={urlLink}>
+        {children}{" "}
+        <Icon
+          icon={IconNames.SHARE}
+          intent={Intent.PRIMARY}
+          size={"0.75rem"}
+          style={{ paddingBottom: "5px" }}
```

Comment:
I think `size={IconSize.STANDARD}` without the padding looks fine, but if you think it's too big, you could change the size to something like `size={IconSize.STANDARD * 0.75}` (and keep the padding).